lib/datasource/docker/index.spec.ts

```diff
@@ -447,6 +447,43 @@ describe(getName(), () => {
       expect(res.releases).toHaveLength(1);
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
+    it('uses quay api', async () => {
+      const tags = [{ name: '1.0.0' }];
+      httpMock
+        .scope('https://quay.io')
+        .get(
+          '/api/v1/repository/node/tag/?limit=100&page=1&onlyActiveTags=true'
+        )
+        .reply(200, { tags, has_additional: false })
+        .get('/v2/')
+        .reply(200, '', {})
+        .get('/v2/node/manifests/1.0.0')
+        .reply(200, '', {});
+      const config = {
+        datasource: id,
+        depName: 'node',
```

Comment:
can we use a real dep name? as i know `node` isn't supported. maybe [`bitnami/redis`](https://quay.io/repository/bitnami/redis)  is a good sample.